rcgen/src/lib.rs

```diff
@@ -139,6 +139,42 @@ struct Issuer<'a, S> {
 	key_pair: &'a S,
 }
 
+impl<'a, S: SigningKey> Issuer<'a, S> {
+	fn new(params: &'a CertificateParams, key_pair: &'a S) -> Self {
+		Self {
+			distinguished_name: &params.distinguished_name,
+			key_identifier_method: &params.key_identifier_method,
+			key_usages: &params.key_usages,
+			key_pair,
+		}
+	}
+}
+
+trait ToDer {
```

Comment:
as I've said previously, I'd prefer a better name for that trait: `SignDer` or something like it.

Reply:
Yup. Do you think the introduction of the trait carries its weight here? I'm not totally convinced...

Reply:
not really tbh. I think having consistent inherent functions is better (traits need to be imported to be used/called, and even if you add inherent functions on top, they still add complexity, which needs to be justified, and in this case i think it's not).